packages/rspack-test-tools/etc/test-tools.api.md

```diff
@@ -4,10 +4,6 @@
 
 ```ts
 
-/// <reference types="../jest.d.ts" />
```

Comment:
I do not know if this is the change of `api-extractor`, it does not matter I think